Gopkg.toml

```diff
@@ -27,7 +27,7 @@ ignored = [ "github.com/ethereum/go-ethereum/ethapi" ]
 
 [[constraint]]
   name = "github.com/ethereum/go-ethereum"
-  version = "v1.8.13"
+  version = "=v1.8.14"
```

Comment:
`=` is needed?

Reply:
i believe that it is a correct workflow. without `=` it is possible to pull higher version accidentally. i had such situations, and `=` just saves time by being pinning version

Reply:
So confusing but you're right:

> Note: When you specify a version without an operator, dep automatically uses the ^ operator by default. dep ensure will interpret the given version as the min-boundary of a range, for example:

> 1.2.3 becomes the range >=1.2.3, <2.0.0